index.bs

```diff
@@ -149,6 +149,12 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
+<h4 id="object-formats">Common Object Formats</h4>
+
+Typically objects will be printed in a format that is suitable for their context. This section describes common ways in which objects are formatted to be most useful in their context.
```

Comment:
Oh, and it may be useful in this section to kind of expand on the idea that these are implementation-specific representations which will eventually end up Printer, which is where they have actual importance.